src/app/Scenes/MyCollection/Components/MyCollectionZeroArtists.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Flex, useScreenDimensions, useSpace } from "@artsy/palette-mobile"
+import { ZeroState } from "app/Components/States/ZeroState"
+import { ModalCarousel } from "app/Scenes/HomeView/Components/ModalCarouselComponents/ModalCarousel"
+import { RouterLink } from "app/system/navigation/RouterLink"
+import { isFontScaleLarge } from "app/utils/accessibility"
+import { useState } from "react"
+import { Image } from "react-native"
+
+export const MyCollectionZeroArtists: React.FC<{}> = () => {
+  const space = useSpace()
+  const { width: screenWidth } = useScreenDimensions()
+  const [isModalVisible, setIsModalVisible] = useState(false)
+
+  return (
+    <Flex>
+      <ModalCarousel
+        isVisible={isModalVisible}
+        toggleModal={(isVisible) => setIsModalVisible(isVisible)}
+      />
+      <ZeroState
+        minHeight={0}
+        showBorder
+        bigTitle="Keep track of artists you collect"
+        subtitle="Discover more about the artists you collect, with latest career news and auction results."
+        image={
+          <Image
+            source={require("images/my-collection-artists-empty-state.webp")}
+            resizeMode="cover"
+            style={{
+              // Avoid making the image too wide on wide screens
+              width: Math.min(screenWidth - 2 * space(2), 600),
```

Comment:
Could setting the max width work as well?

```suggestion
              maxWidth: 600,
```

Reply:
Not really actually, here my goal is the following:
- If the screen is small, the container width should be the full width minus the padding on the sizes
- if the screen is too big, the container width should not be more than 600 pixels, otherwise it would look funny if it's too wide
let me extract this so it's clearer and add some comments to it